components/DataRequirementsViewer.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import { useRecoilState } from 'recoil';
+import { selectedMeasureState } from '../atoms/selectedMeasure';
+import { Loader, ScrollArea, Card, Center, Tabs, Title, Text } from '@mantine/core';
+import dynamic from 'next/dynamic';
+const DynamicReactJson = dynamic(import('react-json-view'), { ssr: false });
+
+/**
+ * Pings DEQM-test-server for the $data-requirements results based on the measure
+ * selected. Updates each time a new measure is selected
+ * @returns JSX for Data Requirements panel on screen
+ */
+export default function DataRequirementsViewer() {
+  const [selectedMeasure] = useRecoilState(selectedMeasureState);
+  const [dataRequirements, setDataRequirements] = useState<fhir4.DataRequirement | null>(null);
+  const [error, setError] = useState<fhir4.OperationOutcome | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const dataRequirementsQuery = '?periodStart=2019-01-01&periodEnd=2019-12-31';
+  useEffect(() => {
+    if (selectedMeasure) {
+      setIsLoading(true);
+      const dataReqUrl = `${process.env.NEXT_PUBLIC_DEQM_SERVER}/Measure/${selectedMeasure}/$data-requirements${dataRequirementsQuery}`;
+      fetch(dataReqUrl)
+        .then(res => res.json())
+        .then(dr => {
+          setDataRequirements(dr);
+          setError(null);
+          setIsLoading(false);
+        })
+        .catch(e => {
+          setError(e);
+          setIsLoading(false);
+        });
+    }
+  }, [selectedMeasure]);
+
+  return (
+    <Card shadow="xl">
+      <Title order={4}>Data Requirements</Title>
+      <Tabs>
+        <Tabs.Tab label="JSON">
+          <ScrollArea style={{ height: 300 }}>{renderJSONResults(isLoading, dataRequirements, error)}</ScrollArea>
+        </Tabs.Tab>
+        <Tabs.Tab label="Filters">
+          <ScrollArea style={{ height: 300 }}>
```

Comment:
Freaking scrollable components and percentage dimensions. I could not figure it out 🤦 . Any tips here? If not, how do you feel about this size?